### ISSUE TITLE
Issues with version 0.17.2

### ISSUE THREAD

**Issue by 9zang** [vrtmrz/obsidian-tagfolder#83]
I love this plugin and thanks to the author for his efforts.
The problem now is that when upgrading this plugin from version 0.17.0 to version 0.17.2 updated today, there is a serious problem, and after clicking the update file to download the update, the obsidian immediately freezes.
After restarting the obsidian, the screen stays on the loading page until you go to the plugin folder to delete the tagfolder plugin, and you can log in to obsidian normally.
After that, re-download versions below 0.17.0 to use normally.
I don't know if there is a conflict with other plugins, hopefully the author will fix this.
Thanks.

**Comment by promethean** [comment]
Agreed - the tag folder plug-in is freezing obsidian.  It's not usable now.  Have turned off all other plugins but still freezes.

Will try using 0.17.0 as suggested.

**Comment by vrtmrz** [comment]
Thank you for opening this issue, and I apologise for your inconvenience! 
Before v0.17.2, link tracing was working excessively. It has been fixed at v0.17.3.
If we are not using LinkFolder, there are no more extra load. And if we are using it, it have been done in lean way.

Could you please try v0.17.3? 

Sorry for a brief reply from mobile!

**Comment by 9zang** [comment]
I just tried version 0.17.3 and the bug has been solved, thank you for solving the problem so quickly! @vrtmrz
